### PR TITLE
Update og, share, roles, death

### PR DESCRIPTION
**Pull request by johnsoncze**
[trello/190] Detail of death politician
[trello/172] Remove LinkedIn share button
[trello/187] Previous roles format
[trello/2] Updated og image

### CODE REVIEW

File: Frontend/src/pages/detail/detail.module.scss [hunk 1]
```diff
@@ -58,7 +58,10 @@
   .fullname {
     @include headerTitle;
     text-transform: capitalize;
-  }
+	}
+	.fulnameDeath {
```
Comment (sauwage) [inline Frontend/src/pages/detail/detail.module.scss:62]: uz parkrat jsem si vsiml, ze se Ti v cssku michaji mezery a taby?

File: Frontend/src/pages/detail/detail.tsx [hunk 2]
```diff
@@ -96,10 +89,16 @@ const Detail: React.FC<Props> = props => {
           </Helmet>
           <div className={styles.heading}>
             <div className={styles.wrapper}>
-              <ProfilePicture src={props.photoUrl} name={props.fullname} customClassName={styles.photo} />
+              <ProfilePicture
+                src={props.photoUrl}
+                name={props.fullname}
+                customClassName={classnames(styles.photo, { [styles.photoDeath]: props.deathYear })}
```
Comment (sauwage) [inline Frontend/src/pages/detail/detail.tsx:95]: ty mezery mezi { a [ jsou uvnitr objektu schvalne?

**Comment by sauwage** [comment]
integraci necham na Tobe, aby sis vsiml komentaru

File: Frontend/src/pages/detail/detail.module.scss (hunk 1 above)
Comment (johnsoncze) [reply Frontend/src/pages/detail/detail.module.scss:62]: Mely by byt jen mezery. Az dodelam nejakou automatiku formatovani pred commitem, tak se to uz stavat nebude.

File: Frontend/src/pages/detail/detail.tsx (hunk 2 above)
Comment (johnsoncze) [reply Frontend/src/pages/detail/detail.tsx:95]: ano

**Pull request merged**